ts/screens/authentication/IdpSelectionScreen.tsx

```diff
@@ -86,10 +88,10 @@ const idps: ReadonlyArray<IdentityProvider> = [
     profileUrl: "https://myid.sieltecloud.it/profile/"
   },
   {
-    id: "spiditalia",
+    id: "spiditaliaid",
```

Comment:
if we call the backend with this ID it fails
`https://app-backend.io.italia.it/login?entityID=spiditaliaid&authLevel=SpidL2`
it muste be **spiditalia**